scss/button/_layout.scss

```diff
@@ -232,6 +233,7 @@
         &::before {
             content: "";
             background-color: currentColor;
+            @include border-radius();
```

Comment:
It seems the design of the default theme does not have a border-radius on the bare buttons ([link](https://github.com/telerik/kendo-ux/blob/master/themes/default/buttons.psd)). This may be intentional for some uses in Kendo UI for jQuery (MediaPlayer / ToolBar?), so maybe it's better to leave it as a variable.